go.mod

```diff
@@ -14,8 +14,87 @@ require (
 	github.com/prometheus/common v0.28.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.2.1
-	k8s.io/api v0.22.1
-	k8s.io/apimachinery v0.22.1
-	k8s.io/apiserver v0.22.1 // indirect
-	k8s.io/client-go v0.22.1
+	k8s.io/api v0.23.1
+	k8s.io/apimachinery v0.23.1
+	k8s.io/apiserver v0.23.1 // indirect
+	k8s.io/client-go v0.23.1
+)
+
+require (
```

Comment:
I tried the bump locally and did not need all this require stuff @dperaza4dustbit 

remove it and re-run `go mod tidy` then `go mod vendor`

if for some reason you can't do this on your Mac without this, then I'll submit an alternative PR

Reply:
Did you move go to 1.7 at the top of go.mod?

Reply:
yep your are right @dperaza4dustbit 

and now looking at it again, I see it now does 1 require blocks ... the top one with direct dependencies and the bottom one with indirect one 

let's leave as is and retest after merging the release PRs